spice/deps/helper_cb.py

```diff
@@ -64,7 +64,6 @@
 if args.clear:
     clipboard.clear()
     clipboard.set_text("")
-    clipboard.store()
```

Comment:
ok you believe - i tested it. if you remove set_text(""), clipboard won't be cleared.